File: frontend-vibe/src/app/models/analysis-info.ts


export default class AnalysisInfo {
  avatarName: string;
  algorithm: string;
  scenario: string;


  constructor(avatarName: string | null = null,
                algorithm: string | null = null,
              scenario: string | null = null,
              ) {
    this.avatarName = avatarName;
    this.algorithm = algorithm;
    this.scenario = scenario;
  }
}

File: VIBEBackend/ReportService/Models/AnalysisInfo.cs
namespace ReportService.Models
{
	public class AnalysisInfo
	{
		public string AvatarName { get; set; }
		public string Scenario { get; set; }
		public string Algorithm { get; set; }

		public AnalysisInfo(string avatarName, string scenario, string algorithm)
		{
			AvatarName = avatarName;
			Scenario = scenario;
			Algorithm = algorithm;
		}
	}
}